scripts/cibuild

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "${GT_SITE_DEBUG}" ]]; then
+    set -x
+fi
+
+if [[ -n "${GIT_COMMIT}" ]]; then
+    GIT_COMMIT="${GIT_COMMIT:0:7}"
+else
+    GIT_COMMIT="$(git rev-parse --short HEAD)"
+fi
+
+function usage() {
+    echo -n \
+"Usage: $(basename "$0")
+Build application for staging or release.
+"
+}
+
+if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
+    if [ "${1:-}" = "--help" ]; then
+        usage
+    else
+
+        docker-compose build gtsite-service gtsite-assets
+        make clean
+
+        echo "Building static asset bundle..."
+        docker-compose run --rm gtsite-assets
+
+        echo "Running tests..."
+        ./scripts/test
+
+        echo "Building geotrellis-service JAR..."
+        docker-compose run --rm --entrypoint "/bin/bash -c" gtsite-service \
+           "unzip -d /srv/data/hillshade/ -o /srv/data/hillshade/hills.zip"
+        docker-compose run --rm gtsite-service assembly
+
+        echo "Building gtsite-service and gtsite-nginx containers"
+        GIT_COMMIT="${GIT_COMMIT}" \
```

Comment:
I think that we can move this and the assembly build above to the `cipublish` script. This will reduce the duration of PR builds, but we'll still be able to test them with `test`.